store/dataStore.test.ts

```diff
@@ -5,89 +5,167 @@ import { useDataStore } from './dataStore';
 const userData = {
   userId: '123',
   userEmail: 'test@email.com',
+  league: ['123456'],
```

Comment:
This should be `leagues` as it's an array.

Reply:
✅ Done!